### ISSUE TITLE
GettingTeamName&IdFlow only pulls 100 teams

### ISSUE THREAD

**Issue by Remrap** [OfficeDev/microsoft-teams-apps-classroom-dropin#2]
The flow GettingTeamName&IdFlow only pulls 100 teams.  My district has over 16,000.  While I could run the flow 160 times, it only pulls the same 100 out causing mass duplication.

**Comment by jok0** [comment]
Ours does the same.  Did you find any way around this?

Thanks

**Comment by Remrap** [comment]
Nope.  Still waiting for the SDS integration to hopefully fix all the limitations.
